netlib/http/request.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, first_line_format, method, scheme, host, port, path, http_ver
 
         self.first_line_format = first_line_format
         self.method = method
-        self.scheme = scheme
+        self.scheme = scheme or b''
```

Comment:
I think I linked the wrong lines. [These](https://travis-ci.org/mitmproxy/mitmproxy/jobs/140021906#L367-L384) are the ones!